config/routes.rb

```diff
@@ -3,5 +3,11 @@
     get 'drivers(.:format)',
         :to => 'api/v1/trucking/drivers#index',
         :defaults => { :format => 'json' }
+    get 'driver_commissions_history(.:format)',
+        :to => 'api/v1/trucking/driver_commissions_history#index',
+        :defaults => { :format => 'json' }
+    get 'driver_commissions_history_by_paid_date(.:format)',
+        :to => 'api/v1/trucking/driver_commissions_history#by_paid_date',
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.